languages/python/exercises/concept/strings/.docs/hints.md

```diff
@@ -0,0 +1,22 @@
+# Hints
+
+### General
+
+- The [Python documentation for `str`][https://docs.python.org/3.7/library/stdtypes.html#text-sequence-type-str] is an overview of the Python `str` type.
```

Comment:
The rendering of this is going to be all off ... there are two ways to do links in markdown, either:

```
The [Pretty Title](https://example.com/link_in_parentheses.html) way.
```
Which renders like: [this](https://youtu.be/dQw4w9WgXcQ).

**OR**


```
The [Pretty Title][indirect-reference] way.

[indirect-reference]: https://example.com/this_will_not_render.html
```

Which renders [the same way][rickroll].

[rickroll]: https://youtu.be/dQw4w9WgXcQ

Either way you've got to be consistent, and you've mixed approaches throughout this file ... maybe double check all your links. Overall the indirect reference method is probably the best choice for keeping links organized, which makes them easier to maintain.

Reply:
... you just RickRolled me

Reply:
(Regardless....) thanks, will standardize the links. :)

Reply:
Addressed.